compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -554,9 +560,6 @@ error.remote.function.in.non.client.object=\
 error.resource.function.in.non.service.object=\
   a resource function allowed in services only
 
-error.resource.function.with.visibility.qualifier=\
```

Comment:
We'll have to remove the DiagnosticCode and usage of the same for this error. 

https://github.com/ballerina-platform/ballerina-lang/blob/f8d7bc4b05a086b4ef01ca0988829624e37ce16b/compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java#L165

Reply:
Fixed.